README.md

```diff
@@ -1,7 +1,5 @@
-# curriculum-databases-projects-template
+# Vet-Clinic
 
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
 
 
 ## Getting Started
```

Comment:
- [OPTIONAL] I think it would be better also to explain to the users how they can get your project locality, and how to clone it,... 🙃 🙃